### ISSUE TITLE
pre-trained models across technologies

### ISSUE THREAD

**Issue by igordot** [jingshuw/SAVERX#8]
Does it make sense to use pre-trained models for other technologies? For example, human immune model is based on 10x data. Can it be used for non-10x datasets? Do you know how much of an effect the dropout rate and coverage (3' or 5' versus full transcript) have?

On a related note, your publication states SAVER-X can only denoise UMI counts. Do you have any recommendations for non-UMI data? Can it perhaps be adjusted in some way?

**Comment by jingshuw** [comment]
SAVER-X can be used across UMI-based technologies. The reason that we can not denoise non-UMI counts is that the technical noise of non-UMI counts is very complicated, making it kind of impossible to denoise with good properties. We have tried some adjustments but have not found a universally working solution
